.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+
+    - name: Update Aptitude cache
```

Comment:
what about other OSes ?

Reply:
This is running on ubuntu.

Reply:
This is not worth testing on several OSes as this is a backend that will ultimately on a single OS. Note we're not using any OS-specific feature in the application either.